src/main/java/robot/subsystems/drivetrain/pure_pursuit/PurePursue.java

```diff
@@ -30,50 +44,74 @@ public PurePursue(Path path) {
     protected void initialize() {
         lastLeftEncoder = drive.getLeftDistance();
         lastRightEncoder = drive.getRightDistance();
-        initAngle = Robot.navx.getAngle();
+        //initAngle = drive.getAngle() + (direction == -1 ? 180 : 0);
+        currentLookahead = path.getWaypoint(0);
+        lastLeftSpeed = direction * drive.getLeftSpeed();
+        lastRightSpeed = direction * drive.getRightSpeed();
     }
 
     // Called repeatedly when this Command is scheduled to run
     protected void execute() {
         updatePoint();
-
+        updateLookaheadInPath(path);
+        /*
+        Main methods:
+            updatePoint()
+            updateLookaheadInPath()
+            closest Point
+         */
 
```

Comment:
Oh, No! The robot will not move!
It looks like you forgot to give the current to the motors.
```suggestion
Robot.drivetrain.setSpeed(getLeftSpeedVoltage(), getRightSpeedVoltage());
```

Reply:
oh sorry i just realized i forgot to commit those changes, will do so tomorrow.